src/modules/release/index.ts

```diff
@@ -75,7 +78,7 @@ export default async (
     .split('/')
 
   // Pachamama v2 requires that version tags start with a 'v' character.
-  const tagText = `v${newVersion}`
+  const tagText = `${displayName ? manifest.name : ''}/v${newVersion}`
```

Comment:
Have you thought about actually using `${manifest.name}@${newVersion}` to name the tag?

Reply:
```suggestion
  const tagText = `${displayName ? manifest.name +"/": ''}v${newVersion}`
```

Reply:
Show! It is looking better with the `@` and follows our patter when deploying version `may-app@1.0.0` @estacioneto 
Also fixed the bug, @gris

Reply:
Your code shows that the tag text is actually `app@v${version}`, or am I mistaken?

Reply:
Nice, agree with the suggestion to remove the `v` when the flag is set and also to update PR's description.

Reply:
I forgot to push the change removing the `v`, my bad! 
Also updated the pr description!